ui/public/config.json

```diff
@@ -63,5 +63,121 @@
   "basicZoneEnabled": true,
   "multipleServer": false,
   "allowSettingTheme": true,
-  "docHelpMappings": {}
+  "docHelpMappings": {
```

Comment:
@fermosan I could be wrong but I think these are generated/auto-generated during build and need not be added to version history. cc @DaanHoogland @shwstppr

Reply:
Another note about package-lock.json - what version of npm do you have? (I don't remember if we should be committing package-lock.json changes?)

Reply:
Maybe this change in config.json was after the rebase ? NPM is at 12.13.0

Reply:
Can you remove the added changes to be same as https://github.com/apache/cloudstack/blob/main/ui/public/config.json#L66 ?

Reply:
Resolved.